hexrd/ui/hexrd_config.py

```diff
@@ -337,6 +340,8 @@ def _attributes_to_persist(self):
             ('_recent_images', {}),
             ('azimuthal_overlays', []),
             ('show_azimuthal_legend', True),
+            ('show_all_colormaps', False),
+            ('limited_cmaps_list', [])
```

Comment:
It seems I've accidentally left a blend of defaults that cause the checkbox to be unchecked but internally it defaults to "show all". Does it make more sense to default to the limited list or all color maps so that people aren't too surprised by the change?